shared/search/helpers.tsx

```diff
@@ -7,14 +6,14 @@ import {debounce} from 'lodash-es'
 const debounceTimeout = 1e3
 
 type OwnProps = {
-  onChangeSearchText: ?(s: string) => void,
-  search: (term: string, service: Types.Service) => void,
-  selectedService: Types.Service,
-  searchResultIds: Array<Types.SearchResultId>,
-  selectedSearchId: ?Types.SearchResultId,
-  onUpdateSelectedSearchResult: (id: ?Types.SearchResultId) => void,
-  onAddUser: (id: Types.SearchResultId) => void,
-  searchResultTerm: string,
+  onChangeSearchText: (s: string) => void | null
```

Comment:
bug in code mod, this should be ((s: string) => void) | null. (bug has been fixed)